client/components/data/query-publicize-connections/index.jsx

```diff
@@ -34,24 +34,29 @@ class QueryPublicizeConnections extends Component {
 }
 
 QueryPublicizeConnections.propTypes = {
-	siteId: PropTypes.number,
+	requestConnections: PropTypes.func,
 	requestingConnections: PropTypes.bool,
-	requestConnections: PropTypes.func
+	selectedSite: PropTypes.bool,
+	siteId: PropTypes.number,
 };
 
 QueryPublicizeConnections.defaultProps = {
```

Comment:
Let's take this opportunity to bulletproof our lifecycle methods. We should update to `componentDidMount` and `componentDidUpdate`

See also: https://gist.github.com/bvaughn/923dffb2cd9504ee440791fade8db5f9